uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/staff-leave-report.component.js

```diff
@@ -116,6 +136,18 @@ define([
         });
     }
 
+    /**
+     * Filters the absence types, keeping only the ones that either have
+     * an entitlement greater than 0 or that allow overuse or accrual requests
+     */
+    function filterAbsenceTypes () {
```

Comment:
`removeAbsenceTypesThatCantBeEntitled` is not the greatest name, but `filterAbsenceTypes` is barely explaining what the function is doing.

Reply:
i'm really not sure how to call it...any name would be describing the full implementation and that would be too far